index.html

```diff
@@ -81,79 +82,135 @@ <h2>Dependencies</h2>
+      <li>The <code>importance</code> attribute will have three states:
         <ul>
           <li>
             <code>high</code> - The developer considers the resource as being high priority.</li>
           <li>
             <code>low</code> - The developer considers the resource as being low priority.
           </li>
           <li>
-            <code>auto</code> - The developer does not indicate a preference. This also serves as the default value if the attribute is not
-            specified.
+            <code>auto</code> - The developer does not indicate a preference. This also serves as the attribute's
+            <a>invalid value default</a> and <a>missing value default</a>.
           </li>
         </ul>
       </li>
-
-      <li>Developers would annotate resource-requesting tags such as
-        <code>script</code> and
-        <code>link</code> using the
-        <code>importance</code> attribute as a hint of the preferred priority with which the resource should be fetched.</li>
-
-      <li>Developers would be able to specify that certain resources are more or less important than others using this attribute.
-        It would act as a hint of the intended priority rather than an instruction to the browser.</li>
     </ul>
 
     <p>With this attribute, the browser should make an effort to respect the developer's preference for the importance of a
       resource when fetching it. Note that this is intentionally weak language, allowing for a browser to apply its own preferences
-      for resource priority or heuristics if deemed important.
+      for resource priority or heuristics if deemed important. See the below section for more information.
     </p>
+
     <p>
       <div class="note">
-        <p>Priority Hints compliment existing browser loading primitives such as preload. Preload is a mandatory and high-priority
-          fetch for a resource that is necessary for the current navigation. Priority Hints can hint that a resource's priority should be lower or higher than its default, and can also be used to provide more granular prioritization to preloads.</p>
+        <p>Priority Hints compliment existing browser loading primitives such as preload. Preload is a mandatory fetch for a resource
+          that is necessary for the current navigation. Priority Hints can hint that a resource's priority should be lower or higher than
+          its default, and can also be used to provide more granular prioritization to preloads.</p>
       </div>
     </p>
+
     <p>This is how we conceptually think about different resource types under the hood in browsers today. It may translate well
       to user-space where different types of content share similar properties.</p>
 
+    <h2 id="effects-of-priority-hints">Effects of Priority Hints</h2>
+
+    <p>Exactly what implementations do with Priority Hints is up to them since much of the resource loading pipeline is currently
```

Comment:
Maybe rephrase? e.g. "Since user agents' resource loading pipeline is largely unspecified, this document doesn't mandate specific mapping between priority hints and request prioritization. However, it does describe some ways in which..."